tests/tests-api.php

```diff
@@ -293,9 +300,9 @@ public function test_get_product_stats() {
 		$this->assertArrayHasKey( 'earnings', $out['products'][0]['stats']['monthly_average'] );
 
 		$this->assertEquals( '60', $out['products'][0]['stats']['total']['sales'] );
-		$this->assertEquals( '229.43', $out['products'][0]['stats']['total']['earnings'] );
+		$this->assertEquals( '229.430000', $out['products'][0]['stats']['total']['earnings'] );
```

Comment:
Numerous rounding issues like this were fixed. From what I gather, PHPUnit 5.x was passing these, possibly due to less strict requirements on decimal places. It was treating `229.43` as equal to `229.430000` and thus passing the test. PHPUnit 7.x fails these. So this is (hopefully) just updating the tests to check against the actual true values rather than rounded ones.